### ISSUE TITLE
Setting up Git: homebrew prompts next steps on intel based mac 

### ISSUE THREAD

**Issue by fadingmorsecode** [TheOdinProject/curriculum#28512]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

Relevant text begins on LINE 74 (lesson-note--warning)

`On an Apple Silicon Mac, you will have an extra step to take. If you look at the terminal output after installing Homebrew, you will see “Installation Successful!”. Further down in the terminal, there will be a section called “Next steps”.`

The instructions of STEP 1 say that the terminal will print a “Next steps” section that is exclusive to Apple Silicon Macs. I am using an intel based Mac and that section is being printed despite what the lesson is indicating. 

<img width="571" alt="Screenshot 2024-07-26 at 6 39 46 AM" src="https://github.com/user-attachments/assets/65983906-d82b-4f44-8a18-0a1d63b456b9">

<img width="257" alt="Screenshot 2024-07-26 at 6 55 46 AM" src="https://github.com/user-attachments/assets/4c5421aa-fee4-4a5b-9ff7-5cb59976824b">

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-setting-up-git

**Comment by MaoShizhong** [comment]
@xandora Any thoughts on this?

**Comment by xandora** [comment]
This feels like something that goes back and forth. It wasn't in the Apple Silicon script, then it was, then it wasn't in the Intel script...

If Intel macOS users are seeing this message, then we can likely just drop this section or remove the note part and just make it more of a generic instructional step.

**Comment by fadingmorsecode** [comment]
My vote is making it more generic and saying to make sure to complete additional steps if present. Kinda just future proofs it if there are switch ups.

**Comment by mao-sz** [comment]
Sorry this took a long time to get back round to. Making the note box generic would be sensible.

### Acceptance criteria

- [ ] Change the note box heading to something more generic e.g. `Potential "Next steps"`
- [ ] Amend the note box contents to be more generic e.g "If after "Installation successful" you see...."

Comment below if you'd like to be assigned to work on this issue. Please do not open a PR unless you have been assigned by a maintainer.

**Comment by fadingmorsecode** [comment]
May I be assigned to work on this please?

**Comment by mao-sz** [comment]
You may :)

**Comment by mao-sz** [comment]
@fadingmorsecode is this still something you wish to work on?

**Comment by fadingmorsecode** [comment]
Yes, very sorry. I will do this today
